docs/udp.rst

```diff
@@ -433,3 +433,131 @@ which we can process further and download::
     cube.download("fancy.tiff", format="GTiff")
 
 See :ref:`datacube_from_process` for more information on :func:`~openeo.rest.connection.Connection.datacube_from_process`.
+
+
+.. _workflow_example:
+
+Working Example: Create EVI Timeseries UDP
```

Comment:
Just asking: any reason to call this a "working" example? The Fahrenheit example above  works too

I would just call this something like

```suggestion
UDP Example: EVI timeseries
```